src/tests/WebApi.OpenAPI.SystemTest/ExtractTypeResult/TypedResultController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace WebApi.OpenAPI.SystemTest.ExtractTypeResult;
+
+[ApiController]
+[Route("typedResult")]
+[Produces("application/json")]
+[Consumes("application/json")]
+public class TypedResultController : ControllerBase
+{
+    [HttpGet]
+    [Route("/withOnlyOnePath")]
+    public Ok<TypedResultExample> TypeResultWithOnlyOnePath(int id)
+    {
+        return TypedResults.Ok(new TypedResultExample("Example"));
+    }
+
+    [HttpGet]
+    [Route("/withNoAnnotation")]
+    [ProducesResponseType(typeof(TypedResultExample), StatusCodes.Status200OK)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    public Results<Ok<TypedResultExample>, BadRequest<ProblemDetails>, NotFound> TypedResultWithNoAnnotation(int id)
```

Comment:
Should the naming be?
- TypedResultWithAnnotation
- TypedResultWithNoAnnotation

Reply:
(also update the routes name)